src/scss/components/_targid.scss

```diff
@@ -283,21 +297,30 @@ div.filmstrip {
       }
 
       > .parameters {
+        margin-bottom: 0;
+        grid-area: parameters;
         opacity: 1;
         transition: opacity 0.3s ease;
+
         p {
           margin: 5px 0;
         }
+
+        // add gap between wrapped elements
+        .form-inline {
+          display: inline-flex;
+          flex-wrap: wrap;
+          gap: 3px;
```

Comment:
Not the best solution. We should fix this in tdp_core so that inline-forms have a gap when wrapping. But for now i think this works and has no side effects